docs/sdks/ui-embed/events.md

```diff
@@ -229,9 +229,3 @@ Triggered when a task is updated:
 | identitySignature            | String (can be null)                  | Anonyme                                                                          |
 | identityAvatarUrl            | String (can be null)                  | https://domain-test.engagement.dimelo.com/assets/default_avatar/thumb-412009.png |
 <br>
-
-### Global Events
-
-#### `smcc:user:disconnected`
-
-Triggered when the user is disconnected from the application or on failed authentication when the experimental `Disable redirection to login when embedded` feature is enabled.
```

Comment:
This will be emitted for omnichannel domains only, but it won't completely disappear. Should we really remove it entirely from the doc?

Reply:
After discussion with Cedric, we'll keep the event reference and just add a mention about it being omnichannel only.